proof_ipa.go

```diff
@@ -555,3 +555,62 @@ type bytesSlice []Stem
 func (x bytesSlice) Len() int           { return len(x) }
 func (x bytesSlice) Less(i, j int) bool { return bytes.Compare(x[i], x[j]) < 0 }
 func (x bytesSlice) Swap(i, j int)      { x[i], x[j] = x[j], x[i] }
+
+// Verify is the API function that verifies a verkle proofs as found in a block/execution payload.
+func Verify(vp *VerkleProof, preStateRoot []byte, postStateRoot []byte, statediff StateDiff) error {
+	// TODO: check that `OtherStems` have expected length and values.
+	fmt.Println(statediff)
```

Comment:
I guess this is being useful now. Pointing out to double-check.

Reply:
nope :)

```suggestion
```